### PR TITLE
Profilepage

### PR DESCRIPTION
**Pull request by ttaannjjaa**

### CODE REVIEW

File: src/pages/ProfilePage.js [hunk 1]
```diff
@@ -0,0 +1,345 @@
...
+              id="petname"
+              data-testid="inputPetName"
+              value={editingCutieNameValue}
+              onChange={event => setEditingCutieNameValue(event.target.value)}
+              onKeyDown={onKeyDown}
+              onBlur={onBlurCutieName}
+              maxLength={30}
+            />
+            <label htmlFor="petName" className="sr-only">
+              Kosename des Katze
```
Comment (celinevo) [inline src/pages/ProfilePage.js:153]: Small grammer error. You probably meant `Kosename der Katze`

File: src/pages/ProfilePage.spec.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    );
+
+    const binicon = screen.getByAltText('bin icon');
+    const updateicon = screen.getByAltText('icon rotate arrow to the left');
+
+    expect(binicon).toBeInTheDocument();
+    expect(updateicon).toBeInTheDocument();
+  });
+
+  it('renders 3 input-Elements for text, 1 input-Element for file and one textarea', () => {
```
Comment (celinevo) [inline src/pages/ProfilePage.spec.js:34]: If your write the whole sentence in lower case the `Element` should be lower case as well :)

**Review by celinevo — APPROVED** [review]
Everything looks good to me! Only two small mistakes :)

File: src/pages/ProfilePage.js (hunk 1 above)
Comment (ttaannjjaa) [reply src/pages/ProfilePage.js:153]: many thanks for finding this 😸

File: src/pages/ProfilePage.spec.js (hunk 2 above)
Comment (ttaannjjaa) [reply src/pages/ProfilePage.spec.js:34]: yes thanks, you are right - i will fix it

**Pull request merged**